Dockerfile

```diff
@@ -41,7 +41,7 @@ COPY . /vzlogger
 
 ARG build_test=off
 RUN cmake -DBUILD_TEST=${build_test} \
-    && make \
+    && make -j $(nproc --all) \
```

Comment:
i'm not sure if this is standard practice,
but i see some danger here, that if `nproc` was ever not available, this line would expand to `make -j` and run an unlimited number of jobs in parallel.
while this is somewhat unlikely to happen on a linux system, as `nproc` is in GNU coreutils. i'd probably still write `nproc --all || echo 1` to provide a failsafe...?

Reply:
never seen this, but makes sense...
Ill also extend the PR description!
Thank you!